services/plenty_of_fish.yml

```diff
@@ -0,0 +1,5 @@
74 3.74 0 0 0-.4-1.78c-.27-.5-.7-.9-1.22-1.13V1.5h10.62a3.8 3.8 0 0 0 1.48-.26c.4-.16.75-.44.99-.8h.14c-1.3 3.7 1.58 2.24 2.44 4.46.53 1.38-.6 2.87-2.01 2.9-1.58.03-2.43-1.18-2.03-2.92.22-.94.73-2.4-1.03-2.55-.85-.09-2.06-.09-3-.09-.65 0-1.11.2-1.4.57-.3.39-.45.98-.45 1.78v6.46h.21c.78 0 1.54-.2 2.22-.59a5.17 5.17 0 0 0 1.77-2.05h.1v6.27h-.1a7.59 7.59 0 0 0-1.81-2 3.51 3.51 0 0 0-2.11-.73h-.28v8.57c0 .82.16 1.46.46 1.9.28.44.69.79 1.16 1v.1h-7.37v-.1Z"/><path d="M102.57 23.53v-.1c.38-.24.68-.6.88-1 .22-.5.33-1.05.32-1.6v-8.04c0-.55-.08-1.1-.23-1.62a1.58 1.58 0 0 0-1-1.06v-.08a10.09 10.09 0 0 0 3.73-1.07c.5-.25.97-.58 1.37-.98h.09v12.85c-.01.54.08 1.09.28 1.6.15.4.42.76.78 1v.1h-6.22Zm17.31-4.3a4.82 4.82 0 0 1-2.76 4.44c-.86.42-1.85.63-2.96.63a17.48 17.48 0 0 1-5.05-.79l1.67-4.6h.09c.27.87.66 1.71 1.16 2.49.44.67.92 1.17 1.45 1.5a3.1 3.1 0 0 0 1.64.5c.69 0 1.22-.2 1.62-.6a1.95 1.95 0 0 0 .58-1.43c.01-.5-.17-.98-.51-1.35-.38-.4-.83-.73-1.32-.98a87.2 87.2 0 0 0-1.69-.87c-.47-.22-.93-.46-1.38-.73-.45-.27-.86-.6-1.22-.99a4.25 4.25 0 0 1-1.16-3.1 4.88 4.88 0 0 1 1.53-3.53c.5-.45 1.1-.8 1.74-1.03a6.56 6.56 0 0 1 2.25-.36 15.5 15.5 0 0 1 3.49.4l-1.17 4.06h-.1a6.96 6.96 0 0 0-.78-1.85c-.3-.48-.68-.9-1.13-1.23-.4-.3-.88-.45-1.37-.45-.6 0-1.09.2-1.44.6a2 2 0 0 0-.39 2.11c.1.24.25.45.45.62.43.4.93.72 1.46.96.59.27 1.2.57 1.82.9.6.3 1.15.65 1.68 1.05a4.48 4.48 0 0 1 1.8 3.63Z"/><path d="M119.51 23.44a3.1 3.1 0 0 0 1.15-2.38V5.01c0-.55-.11-1.09-.34-1.59a2.23 2.23 0 0 0-.9-1.05v-.1a7.3 7.3 0 0 0 1.86-.3 8.64 8.64 0 0 0 1.77-.75c.5-.27.96-.6 1.37-1h.11V10.7c0 .2 0 .41-.03.62l.03.02c.28-.33.6-.69 1-1.08a10.2 10.2 0 0 1 2.64-1.88c.41-.2.87-.31 1.32-.32.8 0 1.43.29 1.87.86.44.58.66 1.32.66 2.22v9.92a3.35 3.35 0 0 0 .99 2.38v.08h-6.04v-.08c.36-.32.66-.7.86-1.13.19-.39.28-.8.28-1.23v-8.47c0-.63-.09-1.1-.26-1.42a.96.96 0 0 0-.85-.52c-.3-.02-.67.09-1.11.31-.48.24-.92.54-1.32.9v9.21c0 .42.1.84.3 1.22.2.42.48.8.81 1.13v.09h-6.16v-.1Zm-94.93-8.81c0 1.4.16 2.56.47 3.51.32.95.76 1.67 1.32 2.16.57.49 1.3.75 2.06.73.68 0 1.35-.17 1.93-.51.61-.34 1.1-.86 1.41-1.48l.18.09a7.44 7.44 0 0 1-1.19 2.65 5.05 5.05 0 0 1-4.18 2.19 4.7 4.7 0 0 1-2.08-.49 5.89 5.89 0 0 1-1.86-1.4 6.66 6.66 0 0 1-1.33-2.32 11.46 11.46 0 0 1-.03-6.5 8.9 8.9 0 0 1 1.3-2.7 6.5 6.5 0 0 1 1.99-1.84c.75-.44 1.6-.67 2.48-.67.9 0 1.79.27 2.52.8a5.58 5.58 0 0 1 1.78 2.14c.44.9.66 1.96.66 3.17l-9.48 2.76v-1.07l5.49-1.54a27.92 27.92 0 0 0-.2-3.16 4.82 4.82 0 0 0-.53-1.67c-.24-.38-.53-.57-.88-.57-.32 0-.62.2-.9.63-.28.42-.5 1.05-.67 1.89-.17.84-.26 1.91-.26 3.2ZM13.96 4.88C11.3.68 7.04 1.25.22 1.25v.09c.52.2.95.59 1.21 1.08.26.47.38 1.04.38 1.72v16.44c.01.61-.11 1.21-.38 1.76-.25.5-.68.9-1.2 1.1v.09h7.13v-.09c-.5-.22-.92-.6-1.17-1.1a3.78 3.78 0 0 1-.4-1.76V16c.54.13 1.08.2 1.62.19a7.66 7.66 0 0 0 6.84-4.3c.47-.97.7-2.1.7-3.41a6.6 6.6 0 0 0-1-3.6ZM9.45 13.6a3.33 3.33 0 0 1-2.96 1.65c-.24 0-.49-.02-.73-.05V7.56l.01-4.63a3 3 0 0 0-.1-.88c1.62 0 2.43.5 3.16 1.19.74.68 1.86 2.68 1.86 5.52a9.4 9.4 0 0 1-1.24 4.85Z"/></svg>
```

Comment:
It has another icon
![image](https://github.com/AdguardTeam/HostlistsRegistry/assets/8361299/363bdbb1-7424-452d-8066-8791a444a931)

Reply:
It is ugly, I made another one.